enum/Analytics.ts

```diff
@@ -8,6 +8,9 @@ export enum EventName {
 	AGENT_TRANSFER_SUCCESSFUL = 'agent_transfer_successful',
 	CHAT_CLOSED_BY_AGENT = 'chat_closed_by_agent',
 	CHAT_CLOSED_BY_TIMEOUT = 'chat_closed_by_timeout',
+	LIVEAGENT_SESSION_EXPIRED = 'live_agent_sesstion_expired',
```

Comment:
Spelling mistake here `sesstion`.

Also, can we add the `CHAT_CLOSED` prefix for these new events? So that it's clear that the actual event that is being sent is the chat closed event